tianshou/env/gym_wrappers.py

```diff
@@ -26,3 +26,28 @@ def __init__(self, env: gym.Env, action_per_branch: int) -> None:
     def action(self, act: np.ndarray) -> np.ndarray:
         # modify act
         return np.array([self.mesh[i][a] for i, a in enumerate(act)])
+
+
+class MultiDiscreteToDiscrete(gym.ActionWrapper):
+    """Gym environment wrapper to discrete action in multidiscrete environment.
+
+    :param gym.Env env: gym environment with continuous action space.
+    """
+
+    def __init__(self, env: gym.Env) -> None:
+        super().__init__(env)
+        assert isinstance(env.action_space, gym.spaces.MultiDiscrete)
+        self.num_dim = env.action_space.shape[0]
+        assert len(
+            set(env.action_space.nvec)
+        ) == 1  # TODO support for different num of actions per dim
+        self.action_per_dim = env.action_space.nvec[0]
+        self.action_space = gym.spaces.Discrete(self.action_per_dim**self.num_dim)
+
+    def action(self, act: np.ndarray) -> np.ndarray:
+        # modify act
+        converted_act = []
+        for i in range(self.num_dim):
+            converted_act.append(act // self.action_per_dim**(self.num_dim - i))
+            act = act % self.action_per_dim**(self.num_dim - i)
+        return np.array(converted_act)
```

Comment:
```python
converted_act = []
for b in reverse(self.bases):
    converted_act.append(act // b)
    act = act % b
return np.array(converted_act).transpose()
```

Reply:
Hi, thanks for your comments, would you define self.baseslike this?
self.bases = [b**(self.num_dim - i) for i, b in enumerate(env.action_space.nvec)]

Reply:
Sorry for some reason my previous comment wasn't showing up. Please see my other comment.

Reply:
`converted_act.append(act // b)` act is an array, so this is not correct. Can you review and correct it?